--- a/autoload/javacomplete.vim	2007-09-26 05:20:38.000000000 +0200
+++ b/autoload/javacomplete.vim	2009-12-25 20:32:07.000000000 +0100
@@ -1746,16 +1746,16 @@
 endfu
 
 fu! s:GetJavaCompleteClassPath()
-  let classfile = globpath(&rtp, 'autoload/Reflection.class')
+  let classfile = globpath(&rtp, 'autoload/Reflection.class', 1)
   if classfile == ''
-    let classfile = globpath($HOME, 'Reflection.class')
+    let classfile = globpath($HOME, 'Reflection.class', 1)
   endif
   if classfile == ''
     " try to find source file and compile to $HOME
     let srcfile = globpath(&rtp, 'autoload/Reflection.java')
     if srcfile != ''
       exe '!' . javacomplete#GetCompiler() . ' -d "' . $HOME . '" "' . srcfile . '"'
-      let classfile = globpath($HOME, 'Reflection.class')
+      let classfile = globpath($HOME, 'Reflection.class', 1)
       if classfile == ''
 	echo srcfile . ' can not be compiled. Please check it'
       endif
@@ -1779,7 +1779,7 @@
 	let b:classpath_jsp .= s:PATH_SEP . path . '/WEB-INF/classes'
       endif
       if isdirectory(path . '/WEB-INF/lib')
-	let libs = globpath(path . '/WEB-INF/lib', '*.jar')
+	let libs = globpath(path . '/WEB-INF/lib', '*.jar', 1)
 	if libs != ''
 	  let b:classpath_jsp .= s:PATH_SEP . substitute(libs, "\n", s:PATH_SEP, 'g')
 	endif
